_docs/concepts/manager.md

```diff
@@ -0,0 +1,39 @@
+---
+category: Concepts
+title: Istio-Manager
+
+parent: Traffic Management
+order: 10
+
+bodyclass: docs
+layout: docs
+type: markdown
+---
+
+The Istio-Manager service is responsible for managing the lifecycle of
+Envoy instances deployed across the Istio service mesh. It acts as a
+discovery service for Envoy, providing service discovery, dynamic route
```

Comment:
linkif to Envoy discovery service doc, also you say 'service discovery' twice here :)